dts/bindings/pwm/maxim,max31790-pwm.yaml

```diff
@@ -3,14 +3,13 @@
 
 description: Maxim MAX31790 6-channel I2C-bus PWM controller
 
-compatible: "maxim,max31790"
+compatible: "maxim,max31790-pwm"
```

Comment:
looks like this PR forgot to add an entry to the migration guide, since `"maxim,max31790"` is now a different beast. Could you please address in a follow-up PR @benediktibk? Thanks!

Reply:
@kartben It is on my todo list, and I suppose the same thing for the rohm,bd8lb600fs and the infineon,tle9104.